### PR TITLE
Macpie/router http channel url check

### PR DESCRIPTION
**Pull request by macpie**

### CODE REVIEW

**Review by michaeldjeffrey — APPROVED** [review]

File: .env-template [hunk 1]
```diff
@@ -21,5 +23,8 @@ ROUTER_CONSOLE_SECRET=XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
 # Max time to wait for uplinks in ms
 ROUTER_FRAME_TIMEOUT=300
 
-# Turn on/off xor filter worker (anything else than true is off)
-ROUTER_XOR_FILTER_WORKER=true
\ No newline at end of file
+# Turn on/off xor filter worker (anything else than true is off, disabled by default)
+ROUTER_XOR_FILTER_WORKER=true
+
+# Turn on/off HTTP integration url checks like DNS, IP... (anything else than false is on, enabled by default)
```
Comment (mfalkvidd) [inline .env-template:29]: Looks to me like this is disabled by default. Comment says enabled by default.

Is it safe to set this setting to false? If it is safe, why does the option to check exist at all?

Comment (macpie) [reply .env-template:29]: This is just the template but you are right it is confusing, I ll update template to be `true` instead

Comment (macpie) [reply .env-template:29]: See https://github.com/helium/router/pull/261

**Comment by disk91** [comment]
This should be reported in the console/.env-template file as the router is usually get as an image and the env file comes from the console environement when following the documentation.

**Pull request merged**